generate_feeds.py

```diff
@@ -0,0 +1,24 @@
+from scrappers.gnius import GniusScrapper
+from scrappers.bpi import BpiScrapper
+import os
+
+def main():
+    feeds_dir = 'feeds'
```

Comment:
beware if the script is launched from a different folder than the current one.
You should get the file's folder and use it as the root folder